paimon-web-server/src/test/java/org/apache/paimon/web/server/controller/JobControllerTest.java

```diff
@@ -0,0 +1,390 @@
+    }
+
+    @Test
+    public void testGetJobStatus() throws Exception {
+        QueryWrapper<ClusterInfo> queryWrapper = new QueryWrapper<>();
+        queryWrapper.eq("cluster_name", "test_cluster");
+        ClusterInfo one = clusterService.getOne(queryWrapper);
+        JobSubmitDTO jobSubmitDTO = new JobSubmitDTO();
+        jobSubmitDTO.setJobName("flink-job-test-get-job-status");
+        jobSubmitDTO.setTaskType("Flink");
+        jobSubmitDTO.setClusterId(String.valueOf(one.getId()));
+        jobSubmitDTO.setStatements(StatementsConstant.selectStatement);
+
+        String responseString = submit(jobSubmitDTO);
+        R<JobVO> r = ObjectMapperUtils.fromJSON(responseString, new TypeReference<R<JobVO>>() {});
+        assertEquals(200, r.getCode());
+
+        String listJobResponseStr = getJobStatus(r.getData().getJobId());
+        R<JobStatusVO> getJobStatusRes =
+                ObjectMapperUtils.fromJSON(
+                        listJobResponseStr, new TypeReference<R<JobStatusVO>>() {});
+        assertEquals(200, getJobStatusRes.getCode());
+        assertEquals(r.getData().getJobId(), getJobStatusRes.getData().getJobId());
+        assertNotNull(getJobStatusRes.getData().getStatus());
+    }
+
+    @Test
+    public void testGetJobStatistics() throws Exception {
+        QueryWrapper<ClusterInfo> queryWrapper = new QueryWrapper<>();
+        queryWrapper.eq("cluster_name", "test_cluster");
+        ClusterInfo one = clusterService.getOne(queryWrapper);
+        JobSubmitDTO jobSubmitDTO = new JobSubmitDTO();
+        jobSubmitDTO.setJobName("flink-job-test-get-job-statistics");
+        jobSubmitDTO.setTaskType("Flink");
+        jobSubmitDTO.setClusterId(String.valueOf(one.getId()));
+        jobSubmitDTO.setStatements(StatementsConstant.selectStatement);
+        String responseString = submit(jobSubmitDTO);
+        R<JobVO> r = ObjectMapperUtils.fromJSON(responseString, new TypeReference<R<JobVO>>() {});
+        assertEquals(200, r.getCode());
+
+        String getJobStatisticsResponseStr =
+                mockMvc.perform(
+                                MockMvcRequestBuilders.get(jobPath + "/statistics/get")
+                                        .cookie(cookie)
+                                        .contentType(MediaType.APPLICATION_JSON_VALUE)
+                                        .accept(MediaType.APPLICATION_JSON_VALUE))
+                        .andExpect(MockMvcResultMatchers.status().isOk())
+                        .andDo(MockMvcResultHandlers.print())
+                        .andReturn()
+                        .getResponse()
+                        .getContentAsString();
+        R<JobStatisticsVO> getJobStatisticsRes =
+                ObjectMapperUtils.fromJSON(
+                        getJobStatisticsResponseStr, new TypeReference<R<JobStatisticsVO>>() {});
+        assertEquals(200, getJobStatisticsRes.getCode());
+        assertEquals(5, getJobStatisticsRes.getData().getTotalNum());
```

Comment:
Please assert the details.

Reply:
Because some subsequent tests need to assert the specific number of jobs, etc., the previous tests need to be executed first. If there is no order, the number of jobs is uncertain.

Reply:
done.